common4j/src/main/com/microsoft/identity/common/java/exception/ClientException.java

```diff
@@ -188,6 +188,10 @@
      */
     public static final String DUPLICATE_COMMAND = "duplicate_command";
 
+    /**
+     * An interactive session is already in progress.
+     */
+    public static final String INTERACTIVE_SESSION_IN_PROGRESS = "interactive_session_in_progress";
```

Comment:
I think that if we're breaking things, we should allow the application author the ability to have their software recover the situation, by either waiting on the current interactive request to complete or by terminating it.  This means that we need to give them a way to discover what it is, and wait on it/kill it.  We don't have to do it all for them, but it shouldn't be impossible.  And we have to rethink this in the context of the broker, where the calculus is completely different, given that we will be serving requests from multiple applications potentially representing different interests.